docs/products/kafka/howto/enable-follower-fetching.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Enable follower fetching in Aiven for Apache Kafka
+sidebar_label: Enable follower fetching
+---
+
+Enabling follower fetching in Aiven for Apache Kafka allows your consumers to fetch data from the nearest replica instead of the leader, optimizing data fetching and enhancing performance.
+
+## Prerequisites
+
+- Aiven for Apache Kafka service version 3.6 or later.
+- [Availability zone (AZ)](#identify-availability-zone) information for your
+  Aiven for Apache Kafka service.
+
+## Identify availability zone
+
+In AWS, availability zone (AZ) names can vary across different accounts. The same
+physical location might have different AZ names in different accounts. To ensure
+consistency when configuring `client.rack`, use the AZ ID, which remains the same
+across accounts.
+
+For mapping AZ names to AZ IDs, see [AWS Knowledge Center article](https://repost.aws/knowledge-center/vpc-map-cross-account-availability-zones) and the [AWS documentation on AZ IDs](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html).
+
+## Enable follower fetching
+
+Enable follower fetching with the [Aiven CLI](/docs/tools/cli):
+
+```bash
+avn service update <service-name> -c follower_fetching={"enabled": true}
```

Comment:
It will actually look like:
`avn service update <service-name> -c follower_fetching.enabled=true`
I confirmed that earlier today

Reply:
Fixed this.